readme.md

```diff
@@ -150,3 +150,54 @@ The best time for the whole team to start is at the "ideation" or "Design Sprint
 stage so that no time is wasted on this on-boarding later on.
 
 Further reading: https://en.wikipedia.org/wiki/Brooks%27s_law
+
+## The lifecycle of an issue
+
+At dwyl we use github labels to help indicate the type, duration and status of an
+issue. The following image outlines the lifecycle of an issue using labels. In
+this example there are 3 actors, the Product Owner (PO), the Scrum Master
+(SM) and the developer (Cleop).
+
+<img src="https://user-images.githubusercontent.com/16775804/35003068-8d01f586-fae3-11e7-9e35-784092b42d62.png" width=350px />
```

Comment:
Thanks for adding @Cleop!
A couple of notes on this:
+ It should be the development team that adds the time estimate (sometimes the SM but best practice is really the dev team)
+ For step 8, I would suggest that you create a fork in the flow - either closing the issue if satisfied (most likely scenario) or _if necessary_ reporting a bug or opening a new issue

Reply:
Worth also adding a link to our contributing process here for anyone who's curious?